synapse/cortex.py

```diff
@@ -401,18 +401,25 @@ def getCoreInfo(self):
         '''
         return await self.cell.getFeedFuncs()
 
-    async def addFeedData(self, name, items, seqn=None):
+    async def addFeedData(self, name, items, *, viewiden=None):
 
-        wlyr = self.cell.view.layers[0]
+        if viewiden is not None:
```

Comment:
Slightly redundant, since cell.getView accepts None and returns the right view.

Reply:
Removed.